lsan.supp

```diff
@@ -0,0 +1 @@
+leak:demux.c
```

Comment:
this will ignore all leaks in demux.c?
doesn't seem wise.

Reply:
Why?